core/src/main/resources/lib/form/hetero-list/hetero-list.js

```diff
@@ -63,7 +63,10 @@ Behaviour.specify(
       menuAlign.split("-"),
       250
     );
-    menuButton._button.classList.add(btn.className); // copy class names
+    // copy class names
+    for (i = 0; i < btn.classList.length; i++) {
+      menuButton._button.classList.add(btn.classList.item(i));
+    }
```

Comment:
Using a traditional for loop rather than `.forEach` because although modern browsers support both, HtmlUnit only supports the former:

> `TypeError`: Cannot find function `forEach` in object